docs/source/installation.md

```diff
@@ -1,45 +1,77 @@
 # Installation
 
-xcube can be installed from a released conda package, or directly from a
-copy of the source code repository.
+## Prerequisites
 
-The first two sections below give instructions for installation using conda,
-available as part of the [miniconda
-distribution](https://docs.conda.io/en/latest/miniconda.html). If installation
-using conda proves to be unacceptably slow, mamba can be used instead (see
-[Installation using mamba](#installation-using-mamba)).
+xcube releases are packaged as conda packages in the
+[conda-forge](https://conda-forge.org/) channel. It is recommended to install
+xcube into a conda environment using the
+[mamba package manager](https://github.com/mamba-org/mamba), which will
+also automatically install and manage xcube’s dependencies.
+You can find [installation instructions for mamba itself
+here](https://mamba.readthedocs.io/en/latest/installation.html), if you
+don’t already have it installed.
 
-## Installation from the conda package
+Note that, while it is technically possible to use the conda package manager
+as an alternative to mamba, we do not recommend this: xcube uses a large
+number of dependencies from the conda-forge channel, and it can take conda
+several hours to resolve these dependencies during installation.
```

Comment:
Please rephrase. This is too negative. 
I fear, this will distract people from using xcube. The list of xcube's dependencies is really not special (Pangeo!). It is just a pitty that the conda channel is so slow, and some dependencies are so large in size. We have also listed some dependency to pin versions, although we actually not import them on our own.

Reply:
Done.